src/components/App/App.constants.js

```diff
@@ -1,3 +1,7 @@
+// action types constants
+export const FINISH_FIRST_VISIT = 'cboard/Speech/FINISH_FIRST_VISIT';
```

Comment:
Should be`'cboard/App/FINISH_FIRST_VISIT'`

Reply:
Oops, copy and paste 😅